### PR TITLE
Cf feature createcomment

### PR DESCRIPTION
**Pull request by ciaranfontein**
- Comments can be created through the input field on every event page.
- Comment components are created through mutation
- Event comments are displayed on event page and loaded from navigation params

### CODE REVIEW

File: client/js/config/constant.js [hunk 1]
```diff
@@ -1 +1,3 @@
 export const APOLLO_CLIENT_ADDRESS = 'http://157.245.224.214:8000/';
```
Comment (siddharthparmar7) [inline client/js/config/constant.js]: can you please rename this to APOLLO_AUTH_ADDRESS?

File: client/js/config/modals.js [hunk 2]
```diff
@@ -22,6 +22,15 @@ export const createViewer = async viewer => {
   }
 };
 
+export const addCommentToEvent = async commentId => {
+  try {
+    //return await
```
Comment (siddharthparmar7) [inline client/js/config/modals.js]: Please explain what is the purpose of this try catch block?

File: client/js/screens/Event/Event.js [hunk 3]
```diff
@@ -75,88 +72,84 @@ const Event = ({
...
+                    try {
+                      const updatedEvent = await updateEvent({
+                        variables: {
+                          title: event.title,
+                          comment: values.comment,
+                          username: user.name,
+                        },
+                      });
+                    } catch (e) {
+                      console.log(e);
```
Comment (siddharthparmar7) [inline client/js/screens/Event/Event.js]: remove this console log.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: client/js/config/modals.js (hunk 2 above)
Comment (ciaranfontein) [reply client/js/config/modals.js]: deleted

File: client/js/config/constant.js [hunk 4]
```diff
@@ -1 +1,3 @@
-export const APOLLO_CLIENT_ADDRESS = 'http://157.245.224.214:8000/';
+export const APOLLO_AUTH_ADDRESS = 'http://157.245.224.214:8000/';
```
Comment (siddharthparmar7) [inline client/js/config/constant.js:1]: Thank you for renaming this const. Make sure you are renaming all the instances of this const. Your branch is not working after renaming. Please make the changes.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: client/js/config/constant.js (hunk 4 above)
Comment (ciaranfontein) [reply client/js/config/constant.js:1]: ok

**Pull request merged**